libraries/Microsoft.Bot.Builder.AI.LUIS/LuisExtensions.cs

```diff
@@ -29,7 +29,7 @@ public static IConfigurationBuilder UseLuisSettings(this IConfigurationBuilder b
             var luisRegion = configuration.GetValue<string>("LUIS_AUTHORING_REGION") ?? configuration.GetValue<string>("region") ?? "westus";
             var environment = configuration.GetValue<string>("environment") ?? Environment.UserName;
             var settings = new Dictionary<string, string>();
-            settings["luis:endpoint"] = $"https://{luisRegion}.api.cognitive.microsoft.com";
+            settings["luis:endpoint"] = configuration.GetValue<string>("luis:endpoint") ?? $"https://{luisRegion}.api.cognitive.microsoft.com";
```

Comment:
>settings["luis:endpoint"] = configuration.GetValue<string>("luis:endpoint") ?? $"https:/ [](start = 12, length = 88)

no I would not do it this way.  do it so it is driven off of the region:
 switch(luisRegion)
 {
  case "chinaeast2":
   endpoint = $"https://{luisRegion}.api.cognitive.azure.cn";
   break;
  default:
   endpoint =  $"https://{luisRegion}.api.cognitive.microsoft.com";
   break;
 }